cartography/intel/azure/key_vaults.py

```diff
@@ -28,7 +54,7 @@ def get_key_vaults_client(credentials: Credentials, subscription_id: str) -> Key
 @timeit
 def get_key_vaults_list(client: KeyVaultManagementClient, regions: list, common_job_parameters: Dict) -> List[Dict]:
     try:
-        key_vaults_list = list(map(lambda x: x.as_dict(), client.vaults.list()))
+        key_vaults_list = list(map(lambda x: x.as_dict(), client.vaults.list_by_subscription()))
         key_vaults_data = []
         for vault in key_vaults_list:
             x = vault['id'].split('/')
```

Comment:
we should move these assignments into a different function like `transform` function implemented other resources in other services.